### PR TITLE
Upstream community operators update (16.06.2021)

### PR DESCRIPTION
**Pull request by mvalarh**
Signed-off-by: Martin Vala <mavala@redhat.com>

### CODE REVIEW

**Comment by lslebodn** [comment]
@mvalarh  do you plan to fix failing tests ?

**Comment by mvalarh** [comment]
Yes i am working on it now.

**Comment by mvalarh** [comment]
It is ready for review

File: upstream/local-test-operator-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      when: run_upstream|bool
+
+  tasks:
+    - name: "Setting basic variables for test tasks"
+      set_fact:
+        doic_skip_file_check: false
+
+    - name: "Running bundle test for operator"
+      include_role:
+        name: test_opertator_bundle
```
Comment (dirgim) [inline upstream/local-test-operator-bundle.yml:34]: typo: test_opertator_bundle -> test_operator_bundle

File: upstream/local-test-operator.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        - always
+      when: run_upstream|bool
+  tasks:
+    - name: "Setting basic variables for test tasks"
+      set_fact:
+        doic_skip_file_check: false
+
+    - name: "Running manifest test for operator"
+      include_role:
+        name: test_opertator_manifest
```
Comment (dirgim) [inline upstream/local-test-operator.yml:41]: typo: test_opertator_manifest -> test_operator_manifest

File: upstream/roles/test_opertator_bundle/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+operator_bundle_src_dir: "/tmp/operator-test/operator-bundle/src"
```
Comment (dirgim) [inline upstream/roles/test_opertator_bundle/defaults/main.yml:1]: typo in naming of the role: test_opertator_bundle -> test_operator_bundle

File: upstream/roles/test_opertator_manifest/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,226 @@
+- name: "Setting basic variables for test operator tasks"
```
Comment (dirgim) [inline upstream/roles/test_opertator_manifest/tasks/main.yml:1]: typo in naming of the role: test_opertator_bundle -> test_operator_bundle

**Review by samvarankashyap — APPROVED** [review]
Looks good to me so far

**Review by dirgim — APPROVED** [review]
LGTM

**Review by lslebodn — APPROVED** [review]
LGTM
lets merge it :-)

**Pull request merged**